lib/page-label/index.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+import box from '../box/index';
+
+var pageLabel = angular.module('guPageLabel', ['guBox']);
+
+pageLabel.directive('guPageLabel', function pageLabelDirective(){
+  return {
+    restrict: 'E',
+    transclude: true,
+    replace: true,
+    template: (el, attrs) => `<gu-box variant="tertiary" class="page-label" ng-class="{ open: isOpen }" ng-mouseover="isOpen = true", ng-mouseleave="isOpen = false">
+      <div class="page-label__content" ng-transclude></div>
+    </gu-box>`
+  };
+});
+
+pageLabel.directive('guPageLabelHover', function pageLabelHoverDirective(){
+  return {
+    restrict: 'E',
+    transclude: true,
+    replace: true,
```

Comment:
I second keeping the custom tag in the HTML. I think it's worth preserving this nice DSL all the way into the browser dev tools!